### PR TITLE
#167697839 remove unused dependencies

### PR DESCRIPTION
**Pull request by raymond42**
**What does this PR do?**
remove unused dependencies

**Description of Task to be completed?**
remove unused dependencies in the package.json file

**How should this be manually tested?**
N/A

**What are the relevant pivotal tracker stories?**
#167697839

### CODE REVIEW

**Review by Jkadhuwa — APPROVED** [review]
Looks good

**Review by placideirandora — COMMENTED** [review]
I see that you have removed almost the unwanted dependencies. But, I would like to know if there is any reason why the '**ejs**' dependency is still there?

**Comment by raymond42** [comment]
> I see that you have removed almost the unwanted dependencies. But, I would like to know if there is any reason why the '**ejs**' dependency is still there?

We can use this for generating template when we are sending emails to people for example like resetting the password. But If there is something else we can use that we can remove but generally that is the reason I left it there

**Comment by placideirandora** [comment]
> > I see that you have removed almost the unwanted dependencies. But, I would like to know if there is any reason why the '**ejs**' dependency is still there?
> 
> We can use this for generating template when we are sending emails to people for example like resetting the password. But If there is something else we can use, we can remove it but generally, that is the reason I left it there

We will eventually need it. So, We can keep it.

**Review by harerakalex — COMMENTED** [review]
Why you didn't remove ejs?

**Comment by raymond42** [comment]
> Why you didn't remove ejs?

I've removed it. Thanks

File: package.json [hunk 1]
```diff
@@ -19,24 +19,15 @@
...
-    "express-jwt": "^5.3.1",
-    "express-session": "^1.15.6",
     "jsonwebtoken": "^8.3.0",
-    "method-override": "^2.3.10",
-    "methods": "^1.1.2",
-    "mongoose": "^5.2.2",
-    "mongoose-unique-validator": "^2.0.1",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "path": "^0.12.7",
```
Comment (mkiterian) [inline package.json]: Node has a path module. Unless there's a valid reason to install this from the npm registry, it should be removed as well

**Review by mkiterian — CHANGES_REQUESTED** [review]
Looks good. Left a comment

File: package.json (hunk 1 above)
Comment (raymond42) [reply package.json]: yes, you are right. I've removed it. thanks

**Pull request merged**